corehq/apps/app_manager/views/modules.py

```diff
@@ -1337,13 +1338,13 @@ def _check_xpath(xpath, location):
                 additional_registry_cases=additional_registry_cases,
                 custom_related_case_property=search_properties.get('custom_related_case_property', ""),
                 inline_search=search_properties.get('inline_search', False),
+                include_related_cases=search_properties.get('include_related_cases', False)
             )
 
     resp = {}
     app.save(resp)
     return JsonResponse(resp)
 
-
```

Comment:
nit: 2 lines between functions is standard

Reply:
fixed with other other renaming - [6fd4217](https://github.com/dimagi/commcare-hq/pull/32587/commits/6fd42170973c08ee8126b8bd5c098e7163c9551a)